app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
```

Comment:
Add an `id` primary key and make `vehicle_id` not the primary_key.

Try adding this line:
```python
id = db.Column(db.Integer, primary_key=True)
```